docs/dev-server-and-tests.md

```diff
@@ -0,0 +1,75 @@
+---
```

Comment:
This was my fault, I forgot to mention updating swagger docs when I wrote this one. Can you add it for me? Thanks

Reply:
Or I can do this in another PR, just mark this as resolved and I would add that later.

Reply:
I'll add this in this PR.